addons/io_scene_gltf2/blender/imp/gltf2_blender_mesh.py

```diff
@@ -323,7 +323,7 @@ def __init__(self, skinning: bool = True, skin_into_bind_pose: bool = True):
     mesh.polygons.foreach_set('loop_total', loop_totals)
 
     for uv_i in range(num_uvs):
-        name = 'UVMap' if uv_i == 0 else 'UVMap.%03d' % uv_i
+        name = 'UVMap' if uv_idx == 0 else 'UVMap.%03d' % uv_idx
```

Comment:
This is still wrong

Reply:
Sorry. It’s the middle of the night and I’m doing this entirely on my phone, so I was simply copy/pasting the same thing without paying attention to the fact that there was a difference on mesh. It’s fixed.

Reply:
Take care ! There is no hurry to manage it on phone in the middle of the night!